README.md

```diff
@@ -104,12 +104,6 @@ Example command:
 - Twitter: [@marwanEg](https://twitter.com/marwaneg)
 - LinkedIn: [marwan-abdelsattar](https://www.linkedin.com/in/marwan-abdelsattar/)
 
-👤 **oguntayomicheal**
-
-- GitHub: [oguntayomicheal](https://github.com/oguntayomicheal)
-- Twitter: [@Oguns_micky](https://twitter.com/Oguns_micky)
-- LinkedIn: [ogunsmicky](https://www.linkedin.com/in/ogunsmicky/)
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍